PCA - Add Loadings Property

s2t2 opened the pull request:
If these are the loadings, can we surface them through a high level property? https://stackoverflow.com/a/42046659/670433

s2t2 committed:
Update _pca.py

If these are the loadings, can we surface them through a high level property?
https://stackoverflow.com/a/42046659/670433

li1127217ye reviewed (changes requested):
Please add documents to explain the meaning of "loadings".

li1127217ye commented on sklearn/decomposition/_pca.py:
```diff
@@ -413,6 +413,10 @@ def __init__(
         return self.n_features_in_
 
+    @property
+    def loadings(self):
```
Please add "_" in the end of function name:
"loadings" -> "loadings_"

s2t2 replied:
Complete!

s2t2 commented:
https://stats.stackexchange.com/questions/143905/loadings-vs-eigenvectors-in-pca-when-to-use-one-or-another

Loadings = Eigenvectors⋅√‾‾‾‾‾‾‾‾‾‾‾Eigenvalues

Loadings = Eigenvectors⋅sqrt(Eigenvalues)

s2t2 committed:
Update _pca.py

Add trailing underscore to follow conventions for properties only available after training
